fullscreen/api/document-fullscreen-enabled-removing-allowfullscreen.sub.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<title>Document#fullscreenEnabled removing allowfullscreen after load and then navigating</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+async_test((t) => {
+  const iframe = document.createElement('iframe');
+  iframe.allowFullscreen = true;
+
+  let i = 0;
+
+  iframe.src = "http://{{domains[www1]}}:{{ports[http][0]}}/fullscreen/api/echo-fullscreenEnabled.html";
```

Comment:
I don't recognize this syntax, is this using a wptserve feature, to make it a cross-origin iframe?